main.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-link').forEach((link) => link.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
```

Comment:
While you did address the changes requested by the previous reviewer, you didn't do so properly. When I click on the hamburger I can now see the mobile menu. However, the links are not taking me to the correct sections on the page. Please fix this.

<img width="664" alt="Screen Shot 2022-06-22 at 8 20 14" src="https://user-images.githubusercontent.com/82426682/175039138-3f88f9c7-66e4-4057-a0d5-5d3c79075934.png">

Additionally, you are not following the template design.

Your design:

<img width="185" alt="Screen Shot 2022-06-22 at 8 20 44" src="https://user-images.githubusercontent.com/82426682/175039237-e588e7c7-2f20-412c-bc63-279fe40cb781.png">

<img width="184" alt="Screen Shot 2022-06-22 at 8 21 02" src="https://user-images.githubusercontent.com/82426682/175039303-28c41e24-4dac-4c16-9ede-193d3cbbddf0.png">

The template design:

<img width="301" alt="Screen Shot 2022-06-22 at 8 21 59" src="https://user-images.githubusercontent.com/82426682/175039499-3c2d9a79-cfe7-4d23-aa75-1826e49a8e83.png">

<img width="303" alt="Screen Shot 2022-06-22 at 8 22 18" src="https://user-images.githubusercontent.com/82426682/175039571-282987d5-0e8b-4ea0-8ce5-b0801539a377.png">

Please fix this.

<img width="845" alt="Screen Shot 2022-06-22 at 8 22 45" src="https://user-images.githubusercontent.com/82426682/175039658-2404183d-32fd-4cd5-89dd-d4b341357fba.png">